DESCRIPTION

```diff
@@ -2,7 +2,21 @@ Package: EpiEstim
 Version: 2.2-0
 Title: EpiEstim: a package to estimate time varying reproduction
     numbers from epidemic curves
-Author: Anne Cori <a.cori@imperial.ac.uk>
+Authors@R: c(person("Anne", "Cori", email = "a.cori@imperial.ac.uk", role = c("aut", "cre")), 
+  person("Simon", "Cauchemez", role = "ctb"),
+  person("Neil M.", "Ferguson", role = "ctb"),
+  person("Christophe", "Fraser", role = "ctb"),
+  person("Thibaut", "Jombart", role = "ctb"),
+  person("Jake", "Stockwin", role = "ctb"),
+  person("Zhian N.", "Kamvar", role = "ctb"),
+  person("Robin", "Thompson", role = "ctb"),
+  person("Rolina", "van Gaalen", role = "ctb"),
+  person("Jonathan A.", "Polonsky", role = "ctb"),
+  person("P. Alex", "Demarsh", role = "ctb"),
+  person("Elisabeth", "Dahlqwist", role = "ctb"),
+  person("Shikun", "Li", role = "ctb"),
+  person("Justin", "Lessler", role = "ctb")
+  )
```

Comment:
Is there a reason for this particular order? 
Also, FWIW, it's possible to add ORCID identifiers and they'll show up in the CRAN description like so (but not necessary)

```r
person("Zhian N.", "Kamvar", "ctb", comment = c(ORCID = "0000-0003-1458-7108"))
```

Reply:
not really a reason for order, but have not reordered so that first is me as main contributor and maintainer, second is by alpheabetical order all those who contributed to EpiEstim 1.0 and then third is by alphabetical order all those who contributed to EpiEstim 2.2

Reply:
I have added the ORCIDs of all people I could find